src/styles/variables.scss

```diff
@@ -0,0 +1,13 @@
+$cell-size: 100px;
+$col-count: 7;
```

Comment:
Consider using a more descriptive variable name like `$calendar-col-count` to improve readability.